_data/locales/ja.yml

```
@@ -0,0 +1,31 @@
+ja:
+  locale_name: 日本語
+  nav:
+    about: このサイトについて
+    contribute: 貢献する
+  index:
+    lead: オープンソースソフトウェアはあなたのような人々によって作られています。プロジェクトを立ち上げて大きくしていく方法を学びましょう。
+    opensourcefriday: 今日は金曜日です！少しの時間を、あなたが使っていて愛着を持っているソフトウェアへの貢献に使いましょう。
+  article:
+    table_of_contents: 目次
+    back_to_all_guides: すべてのガイドに戻る
+    related_guides: 関連するガイド
+  footer:
+    contribute:
+      heading: 貢献する
+      description: 提案を行いたいですか？このコンテンツはオープンソースです。改善の手助けをして下さい。
+      button: 貢献する
+    subscribe:
+      heading: 情報を受け取る
+      description: GitHubの最新のオープンソースのコツや資料を受け取りましょう。
```

```suggestion
      description: GitHub の最新のオープンソースに関するヒントや情報を知らされる最初の人になりましょう。
```
言い回し修正しました。
あと試しに英単語の間にスペースいれてみました

---

ここの日本語訳ですが、"Be the first to hear about ..."は確かに「最初の人になりましょう」という意味もあると思うのですが、このdescriptionはメールで最新の情報を購読するメニューの所に表示されるものなので、文脈的には「…をすぐに受け取りましょう」とか「…をすぐに受け取れるようにしましょう」みたいなニュアンスがより近いかなと思いました。いかがでしょう？

---

なるほど…！確かにそうですね。
コメントいただいた翻訳の方が良いと思います！

---

Fixed 6c63f1a